docker/docker-compose.yml

```diff
@@ -169,37 +169,34 @@ services:
       timeout: 3s
       retries: 3
     volumes:
-      - dbdata:/var/lib/postgresql/data
+      - dbdata_pg13:/var/lib/postgresql/data
```

Comment:
I wonder if we should update this to match the previous volume setup:
`- ${DB_DATA_VOLUME:-dbdata_pg16}:/var/lib/postgresql/data`
It would need an ENV, and I think we'd want a different name for the ENV.  Maybe `DB_DATA_VOLUME_PREVIOUS`.  That way we won't accidentally use the same volume, maybe?

Reply:
Or, maybe my memory is shaky on this.  Do we _intend_ to use the same volume?  If so, then it should mirror what's in `db`.

Reply:
The intention is use a new volume for pg16 and leave the current data volume untouched.

Good point about the DB_DATA_VOLUME. I wonder if we can just remove that var and suggest people use a `docker-compose.override.yaml` file for customizations rather than an env variable.

Reply:
Hmm, maybe, though I think there was some issue with the way volumes/arrays are overlayed that made that difficult.

Reply:
Yes I think you would need to re-declare the whole volumes array in the override, which isn't ideal but should work. Is it your impression folks are actually using the `$DB_DATA_VOLUME`? Perhaps the override is an okay work-around if this var isn't commonly used

Reply:
I'm using it, I think others are too.  I use it so my db is stored on my local machine instead of a docker volume.

Reply:
got it, I'll use the env vars as you suggest and add that to the migration instructions